.github/workflows/lua-doc.yml

```diff
@@ -4,51 +4,45 @@ on: push
 
 jobs:
   generate-lua-docs:
-    name: "Generate lua docs"
+    name: Generate lua docs
     runs-on: windows-latest
 
     steps:
       - uses: actions/checkout@v3.3.0
-
-      - name: Print GitHub context
-        run: echo "$GITHUB_CONTEXT"
-        env:
-          GITHUB_CONTEXT: ${{ toJson(github) }}
-
-      - name: "Generate lua docs with lua-language-server"
-        shell: cmd
+      - name: Setup GitHub
         run: |
-          ${{ github.workspace }}\\.building\\lua-language-server-3.6.18-win32-x64\\bin\\lua-language-server.exe --logpath \"${{ github.workspace }}\\docs\\lua-docs\" --configpath \"${{ github.workspace }}\\docs\\lua-docs\\lua-language-server-config.lua\" --doc \"${{ github.workspace }}\\mods\\noita-mp\\files\\scripts\"
-          cd ${{ github.workspace }}\\docs\\lua-docs\\
-          DEL /f *.log
+          echo "$GITHUB_CONTEXT"
           # https://api.github.com/users/github-actions%5Bbot%5D
           git config --global user.name "github-actions[bot]"
           git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git add .
-          git commit -m "Updated generated lua docs"
-          git push origin
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
 
-      - name: "Fix wrong paths in lua docs"
-        shell: pwsh
+      - name: Generate lua docs with lua-language-server
+        shell: cmd
+        run: |
+          ${{ github.workspace }}\\.building\\lua-language-server-3.6.18-win32-x64\\bin\\lua-language-server.exe ^
+            --logpath \"${{ github.workspace }}\\docs\\lua-docs\" ^
+            --configpath \"${{ github.workspace }}\\docs\\lua-docs\\lua-language-server-config.lua\" ^
+            --doc \"${{ github.workspace }}\\mods\\noita-mp\\files\\scripts\" 
+          cd ${{ github.workspace }}\\docs\\lua-docs\\ 
+          del /f *.log
```

Comment:
Not sure.. Looks like it wasn't working, because I can see log files in `${{ github.workspace }}\\docs\\lua-docs\\`. Maybe add it to `.gitignore`?

Reply:
The branch I gave a link to used the version with commented out del line because I checked the logs